gateway/run.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/sh
 set -eu
-. /opt/.env
```

Comment:
This breaks the gateway, since `GRAPH_NODE_GRAPHQL` & `GATEWAY_API_KEY` are no longer set